docs/core/diagnostics/microsoft-diagnostics-netcore-client.md

```diff
@@ -1,244 +1,64 @@
 
 ## Install
 
-You can acquire [Microsoft.Diagnostics.NETCore.Client](https://www.nuget.org/packages/Microsoft.Diagnostics.NETCore.Client/) by adding a `PackageReference` to your project. The package is hosted on `NuGet.org`. 
+You can acquire [Microsoft.Diagnostics.NETCore.Client](https://www.nuget.org/packages/Microsoft.Diagnostics.NETCore.Client/) by adding a `PackageReference` to your project. The package is hosted on `NuGet.org`.
 
-## Use Microsoft.Diagnostics.NETCore.Client to write your own diagnostic tools
+## Write your own diagnostic tool
 
 These samples show how to use Microsoft.Diagnostics.NETCore.Client library. Some of these examples also show parsing the event payloads by using [TraceEvent](https://www.nuget.org/packages/Microsoft.Diagnostics.Tracing.TraceEvent/) library.
 
 ### Attach to a process and print out all the runtime GC events in real time to the console
 
 This snippet shows how to start an EventPipe session using the [.NET runtime provider](../../fundamentals/diagnostics/runtime-events.md) with the GC keyword at informational level and how to use the `EventPipeEventSource` class provided by the [TraceEvent library](https://www.nuget.org/packages/Microsoft.Diagnostics.Tracing.TraceEvent/) to parse the incoming events and print their names to the console in real time.
 
-```cs
-using Microsoft.Diagnostics.NETCore.Client;
-using Microsoft.Diagnostics.Tracing.Parsers;
-using System.Collections.Generic;
-
-public void PrintRuntimeGCEvents(int processId)
-{
-    List<EventPipeProvider> providers = new List<EventPipeProvider>()
-    {
-        new EventPipeProvider("Microsoft-Windows-DotNETRuntime",
-            EventLevel.Informational, (long)ClrTraceEventParser.Keywords.GC)
-    };
-
-    DiagnosticsClient client = new DiagnosticsClient(processId);
-    using (EventPipeSession session = client.StartEventPipeSession(providers, false))
-    {
-        var source = new EventPipeEventSource(session.EventStream);
-
-        source.Clr.All += (TraceEvent obj) => {
-            Console.WriteLine(obj.ToString());
-        };
-
-        try
-        {
-            source.Process();
-        }
-        catch (Exception e)
-        {
-            Console.WriteLine("Error encountered while processing events");
-            Console.WriteLine(e.ToString());
-        }
-    }
-}
-```
+:::code language="csharp" source="snippets/Microsoft.Diagnostics.NETCore.Client/PrintRuntimeGCEvents.cs":::
```

Comment:
Put these in a `snippets/Microsoft.Diagnostics.NETCore.Client/csharp` folder. This way we can add other languages such as `vb`.